hood/crypto_trading/_protocols.py

```
@@ -1,47 +1,45 @@
-from typing import Protocol, Dict, Optional, TYPE_CHECKING
+from typing import Protocol, Dict, Optional, TYPE_CHECKING, Union, TypeVar, Type, Tuple
 
 from . import constants as _constants
 
 if TYPE_CHECKING:
+    from _typeshed import DataclassInstance  # noqa
```

Ruff (0.14.10)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)